scripts/commands/updateApiDocs.ts

```diff
@@ -100,11 +92,7 @@ zxMain(async () => {
   );
 
   const artifactFolder = pkg.ciArtifactFolder();
-  if (await pathExists(artifactFolder)) {
-    console.log(`Skip downloading sources for ${pkg.name}:${pkg.version}`);
-  } else {
-    await downloadCIArtifact(pkg.name, args.artifact, artifactFolder);
-  }
+  await loadCIArtifact(pkg, artifactFolder);
```

Comment:
I think `download` makes more sense as a name here (`load` makes me think you're loading it into memory).

Reply:
I changed it back to `download`. I agree it makes more sense. Given that I was using the same name for a helper function, the result is changing `loadCIArtifact` and `downloadCIArtifact` to `downloadCIArtifact` and `downloadFromBox` respectively.